rero/rero-ils
# subscriptions: display message when patron has pending subscription

## How to test?

!!! use `rero/zan-US1297-subscriptions` for RERO-ILS-UI

- Assign a patron_type requiring a subscription to patron (Monnet, Philippe). A new subscription fee will be created
- In the public interface, login as this patron (reroilstest+philippe@gmail.com) and display the profile of this patron. An alert box should be displayed to alert there is a pending subscription.

![image](https://user-images.githubusercontent.com/10031585/78679982-33cde280-78eb-11ea-88e1-883c243d1364.png)

## Code review check list
